draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1118,9 +1118,10 @@ arbitrary string to a point on an elliptic curve.
 
 Many cryptographic protocols require a procedure that encodes an arbitrary input,
 e.g., a password, to a point on an elliptic curve. This procedure is known
-as hashing to an elliptic curve. Prominent examples of cryptosystems that
-hash to elliptic curves include password-authenticated
-key exchanges {{BM92}} {{J96}} {{BMP00}} {{p1363.2}}, Identity-Based
+as hashing to an elliptic curve, where the hashing procedure provides collision
+resistance over the set of inputs and produces an output point for which the
```

Comment:
`producing a point for which the DL is difficult to find` is not a property of the hash-to-curve function, it is a property of the ccurve itself (or the (sub)group of points).

Better to say that one application of this hash function is for generating points on the curve such that _the caller_ cannot know its discrete log.

Reply:
Your proposed text seems to match what's written here?

Reply:
(Chatted offline and applied a change)